spec/rails_helper.rb

```diff
@@ -1,4 +1,4 @@
-ENV['RAILS_ENV'] = 'test'
+ENV['RAILS_ENV'] ||= 'test'
```

Comment:
Now this is a ||= again. Did we ever figure out why it was changed before? Just curious...